keyboards/peej/lumberjack/rules.mk

```diff
@@ -16,5 +16,3 @@ NKRO_ENABLE = no            # Enable N-Key Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
```

Comment:
This shouldn't be removed.
```suggestion
AUDIO_ENABLE = no           # Audio output

LAYOUTS = ortho_5x12
```

Reply:
If I don't remove it then the CI checks against all ortho 5x12 layouts and a bunch of user layouts fail to build.

I'm not sure if this is something I can solve.

Reply:
It isn't something you need to. User keymaps are generally meant to be maintained by them.

Reply:
So it is fine that the CI fails one some user layouts?

I have pushed the unremoval of this :)

Reply:
Yeah, it's fine.  Mostly, we're looking for the  default and via keymaps to compile fine (as well as any included keymaps).  If community layouts fail, that's okay.